app-modules/form/src/Filament/Blocks/FormFieldBlockRegistry.php

```diff
@@ -10,6 +10,7 @@ class FormFieldBlockRegistry
     public static function get(): array
     {
         return [
+            StudentEmailFormFieldBlock::class,
```

Comment:
I feel that we should try and keep out naming consistent throughout the app when it comes to working with both Students and Prospects at the same time.

What do you think about renaming this to Educatable rather than Student?

Reply:
For sure

Reply:
We should keep the user-presented label as "Student email", though, right?

Reply:
Yeah that probably makes sense.